common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -166,10 +168,6 @@ public static void submitSilent(@SuppressWarnings(WarningType.rawtype_warning) @
     public static FinalizableResultFuture<CommandResult> submitSilentReturningFuture(@SuppressWarnings(WarningType.rawtype_warning) @NonNull final BaseCommand command) {
 
         final String methodName = ":submitSilent";
-        Logger.verbose(
```

Comment:
Why remove this line?

Reply:
thanks for pointing out, will alter the silent request logging, since some command might not get logged (the ones which are returned right away).

Reply:
replaced it with more structured logging.